core/src/main/java/org/eclipse/dash/licenses/http/HttpClientService.java

```diff
@@ -67,7 +68,16 @@ public int post(String url, String contentType, String payload, Consumer<String>
 					Thread.sleep(1000 * tries);
 					continue;
 				}
-				
+				if (response.statusCode() == 429 && tries++ < maxRatelimitRequestsPerExecure) {
```

Comment:
I'm thinking that reusing _tries_ could result in odd-to-explain behaviour. Five 502s, followed by a 429 means that we stop trying. I think that it makes sense, but is that the behaviour that we want to have?